cmd/upgrade/accountroles/cmd.go

```diff
@@ -135,7 +135,12 @@ func run(cmd *cobra.Command, argv []string) error {
 
 	roleARN, err := awsClient.GetAccountRoleARN(prefix, role.Name)
 	if err != nil {
-		r.Reporter.Errorf("Failed to get account roles ARN: %v", err)
+		if args.hostedCP {
+			r.Reporter.Errorf("Failed to get hosted CP account roles ARN: %v", err)
+		} else {
+			r.Reporter.Errorf("Failed to get classic account roles ARN: %v. "+
+				"To upgrade hosted CP account roles use the '--hosted-cp' flag", err)
```

Comment:
is this correct, for classic account roles?

Reply:
The assumption was that the user want to upgrade HCP roles, but forgets the flag, and then we guide them.
Do you find it confusing? Maybe I can improve the wording here (it is possible they aren't aware of HCP at all)

Reply:
I wonder if we should alert both sides then